bin/cmsl1t_batch_status

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import click
+import click_log
+import glob
+import logging
+import pandas as pd
+import re
+import os
+import sys
+from tabulate import tabulate
+import shutil
+
+from cmsl1t.batch import Batch, Status, condor_status, lsf_status
+
+if sys.version_info[0] < 3:
+    from StringIO import StringIO
+else:
+    from io import StringIO
+
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+
+def update_job_status(row):
+    batch = row.batch
+    batch_id = row.batch_id
+    job_path = row.output_folder
+    job_status = row.status
+    if job_status == Status.FINISHED:
+        return job_status
+    else:
+        # check if output files exist
+        if has_output_files(job_path):
+            return 'FINISHED'
+
+        batch_status = get_batch_status(batch, batch_id)
+        if batch_status:
```

Comment:
In the present implemantion of `get_batch_status` there's no attempt to catch exceptions raised, so that function would fail in such cases and the exception raised for the user (which is probably correct behaviour).  But since that function always returns something, is this check necessary?  And if that function did not return something, or if that object would cast to False in this check, would the right status to return here be `Failed` or `Unknown`, which seems to be what the function itself currently defaults to?